src/instrument.js

```diff
@@ -420,7 +420,8 @@ export default function instrument(monitorReducer = () => null, options = {}) {
     );
   }
 
-  return createStore => (reducer, initialState, enhancer) => {
+  return createStore => (reducer, initialState, enhancer, disabled) => {
```

Comment:
Hmm. I don’t feel good about enforcing a very specific meaning that only makes sense for this particular use case to a generic `createStore` signature.